compiled_starters/ruby/app/server.rb

```diff
@@ -0,0 +1,24 @@
+require 'socket'
+
+def start_server
+  udp_socket = UDPSocket.new
+  udp_socket.bind('127.0.0.1', 2053)
+
+  # You can use print statements as follows for debugging, they'll be visible when running tests.
+  print("Logs from your program will appear here!")
+
+  # Uncomment this block to pass the first stage
+  #
+  # loop do
+  #   begin
+  #     buf, source = udp_socket.recvfrom(512)
```

Comment:
Can we rename `buf` to something like `data` or `message`? Seems more descriptive for this case. For example, https://ruby-doc.org/stdlib-1.9.1/libdoc/socket/rdoc/Socket.html#method-i-recvfrom uses `data` in their example